src/main/java/net/openhft/chronicle/jlbh/TeamCityHelper.java

```diff
@@ -13,11 +14,6 @@
     private TeamCityHelper() {
     }
 
-    public static void histo(@NotNull String name, @NotNull Histogram histo, @NotNull PrintStream printStream) {
-        double[] percentages = Histogram.percentilesFor(histo.totalCount());
-        printPercentiles(name, printStream, percentages, histo.getPercentiles());
-    }
-
```

Comment:
This didn't seem to be used anywhere?

Reply:
Could be used somewhere else. I thought it might be useful, but can't remember if it is used